src/charts/bar.rs

```diff
@@ -162,6 +162,7 @@ pub struct BarChartProps {
 /// - `class_grid_labels`: &[str] (default: `"dx-grid-labels"`): The HTML element `class` of the
 /// group of grid labels.
 #[allow(non_snake_case)]
+#[component]
```

Comment:
Since the component is staking a struct (`BarChartProps`), we don't really need the component macro I think, same for the other components

Reply:
ah, you're right. just checked and it works fine without. was mislead by the compiler error

Component<_> is not implemented for fn(LineChartProps) -> std::option::Option<dioxus_core::nodes::VNode> {LineChart}

but adding the signal feature must have fixed that too.

have updated